### DISCUSSION TITLE
WriteTimeout not set

### DISCUSSION THREAD

**Discussion by sospartan** [pocketbase/pocketbase#182] [Q&A] [answered]
https://github.com/pocketbase/pocketbase/blob/0739e90ff24fd585a0210ab10651caea8ae176d4/cmd/serve.go#L80
Seems the `WriteTimeout`  is turn off on purpose. What is `sse`?

**Comment by ganigeorgiev** [comment, marked as answer]
SSE is short for server-sent events and it is used in PocketBase for the realtime subscriptions.

It is turned off by default until we find a better way to manage individual handler timeouts without hijacking every request connection.

This unfortunately is common go "issue" (see https://github.com/golang/go/issues/16100) and if you really need to set a read/write timeouts, my recommendation is to use a reverse proxy nginx/apache and control the timeouts per location/path.

**Reply by aasutossh** [reply]
The issue you mentioned was closed @ganigeorgiev . Maybe we can continue with this.

**Reply by ganigeorgiev** [reply]
@aasutossh It is already implemented as part of the ongoing refactoring. It will be available with the next v0.23.0 release (_there are no ETAs yet_).
